locales/en-US/messages.properties

```diff
@@ -1 +1,9 @@
+unable_to_contact_login_server=Unable to contact login server.<br/> Refresh the page to try again.
+loading_message=Loading&hellip;
+your_project=Your Projects
+log_out=Log Out
+sign_in=Sign in
+or=or
+sign_up=Sign Up
+hi_username=Hi, {name}
```

Comment:
In English- isn't it possible that other languages the appropriate greeting starts with the username? Remember, this is localization, not just translation. We're the only people that see this (translators don't, right?), but I still think it helps to be conscious of this so that we can break strings up appropriately based not just on English structure but semantics of the string. You don't need to change this specific example, but keep it in mind when pontoon-ifying future pages.